index.html

```diff
@@ -0,0 +1,240 @@
+                </div>
+            </div> 
+            <br><br><br>
+            <div >
+                <div class="comment-image"><img src="img/girl.png" class="reaction">&nbsp; &nbsp; &nbsp;Natsuki-Nata</div>
+                <div class="comments-text">
+                    <p class="comment-text">"What do you want to be when you grow up?"
+                        "A mermaid.".......me, literally, me. </p>
+                    <p class="comment-text">My kid is gonna say.  I WANNA BE A FREAKING BLACK HAT  HACKER</p>
+                    <p class="reaction"><img src="img/like.png" class="reaction">&nbsp; &nbsp; &nbsp;<img src="img/dont-like.png" class="reaction">&nbsp; &nbsp; &nbsp; Reply</p>
+                </div>
+            </div> 
+            <br><br><br>
+            <div >
+                <div class="comment-image"><img src="img/girl.png" class="reaction">&nbsp; &nbsp; &nbsp;Natsuki-Nata</div>
+                <div class="comments-text">
+                    <p class="comment-text">"What do you want to be when you grow up?"
+                        "A mermaid.".......me, literally, me. </p>
+                    <p class="comment-text">My kid is gonna say.  I WANNA BE A FREAKING BLACK HAT  HACKER</p>
+                    <p class="reaction"><img src="img/like.png" class="reaction">&nbsp; &nbsp; &nbsp;<img src="img/dont-like.png" class="reaction">&nbsp; &nbsp; &nbsp; Reply</p>
+                </div>
+            </div>
+            <br><br><br> 
+            <div >
+                <div class="comment-image"><img src="img/girl.png" class="reaction">&nbsp; &nbsp; &nbsp;Natsuki-Nata</div>
+                <div class="comments-text">
+                    <p class="comment-text">"What do you want to be when you grow up?"
+                        "A mermaid.".......me, literally, me. </p>
+                    <p class="comment-text">My kid is gonna say.  I WANNA BE A FREAKING BLACK HAT  HACKER</p>
+                    <p class="reaction"><img src="img/like.png" class="reaction">&nbsp; &nbsp; &nbsp;<img src="img/dont-like.png" class="reaction">&nbsp; &nbsp; &nbsp; Reply</p>
+                </div>
+            </div> 
+            <br><br><br>
+            <div >
+                <div class="comment-image"><img src="img/girl.png" class="reaction">&nbsp; &nbsp; &nbsp;Natsuki-Nata</div>
+                <div class="comments-text">
+                    <p class="comment-text">"What do you want to be when you grow up?"
+                        "A mermaid.".......me, literally, me. </p>
+                    <p class="comment-text">My kid is gonna say.  I WANNA BE A FREAKING BLACK HAT  HACKER</p>
+                    <p class="reaction"><img src="img/like.png" class="reaction">&nbsp; &nbsp; &nbsp;<img src="img/dont-like.png" class="reaction">&nbsp; &nbsp; &nbsp; Reply</p>
+                </div>
+            </div> 
```

Comment:
please use a for loop instead of writing the same thing again and again to repeat a specific block of code. it will make your code space more professional and helps to alter the flow of the program.